framework/packages/abstract-app/Cargo.toml

```diff
@@ -11,7 +11,7 @@ crate-type = ["cdylib", "rlib"]
 
 
 [features]
-interface-macro = ["dep:abstract-interface", "dep:cw-orch"]
+interface-macro = ["dep:abstract-interface", "dep:cw-orch", "abstract-client"]
```

Comment:
Why does this need abstract-client?

Reply:
> Why does this need abstract-client?

Because of `DependencyCreation` trait

Reply:
Ah yeah, should be only for `test-utils` right

Reply:
I think it's only needed in `test-utils`